src/main/java/com/microsoft/sqlserver/jdbc/SQLServerResource.java

```diff
@@ -540,7 +540,8 @@ protected Object[][] getContents() {
         {"R_serverError", "An error occurred during the current command (Done status {0}). {1}"},
         {"R_ManagedIdentityTokenAcquisitionFail", "Failed to acquire managed identity token. Request for the token succeeded, but no token was returned. The token is null."},
         {"R_AmbiguousRowUpdate", "Failed to execute updateRow(). The update is attempting an ambiguous update on tables \"{0}\" and \"{1}\". Ensure all columns being updated prior to the updateRow() call belong to the same table."},
-        {"R_InvalidSqlQuery", "Invalid SQL Query: {0}"}
+        {"R_InvalidSqlQuery", "Invalid SQL Query: {0}"},
+        {"R_InvalidScale", "Scale of input value is larger than the maximum allowed by SQL Server (38)."}
```

Comment:
maybe we don't need to explicitly say 38 here? just max is enough

Reply:
Sure, I can remove that.